src/components/EntityInfo/EntityHeader.vue

```diff
@@ -8,16 +8,27 @@
 
       <!-- Subtitle -->
       <div>
-        <span id="business-description">{{ businessDescription }}</span>
+        <span id="business-description">{{ businessDescription }} </span>
+        <span id="active-util-wrap" class="ml-3"  v-if="isInLimitedRestoration">
+          <span id="active-util" class="ml-3">Active until {{ getLimitedRestorationActiveUntil || 'Unknown' }}</span>
+        </span>
+      </div>
 
-        <span id="limited-restoration" class="ml-3" v-if="isInLimitedRestoration">
-          <span class="ml-3">Active until {{ getLimitedRestorationActiveUntil || 'Unknown' }}</span>
-          <v-chip class="primary mt-n1 ml-3 pointer-events-none font-weight-bold"
+      <div id="historical" class="mt-2" v-if="isHistorical">
+          <v-chip
+            id="historical-chip"
+            class="primary mt-n1 pointer-events-none font-weight-bold"
+             small label text-color="white">HISTORICAL</v-chip>
+          <span class="font-14 mx-3">{{ getReasonText || 'Unknown Reason' }}</span>
+      </div>
+      <div id="multiple-badges" class="mt-2" v-else>
+        <span id="limited-restoration" v-if="isInLimitedRestoration">
+          <v-chip class="primary mt-n1 pointer-events-none font-weight-bold"
             small label text-color="white">LIMITED RESTORATION</v-chip>
         </span>
 
         <span id="authorized-to-continue-out" v-if="isAuthorizedToContinueOut">
-          <v-chip class="primary mt-n1 ml-3 pointer-events-none font-weight-bold"
+          <v-chip :class="dynamicClassForBadges"
```

Comment:
There's another way to do this that you may prefer. (Personally I generally prefer classnames in the template, not in the script.)

```
class="primary mt-n1 pointer-events-none font-weight-bold"
:class="isInLimitedRestoration ? 'ml-3' : ''"
```

or

```
class="primary mt-n1 pointer-events-none font-weight-bold"
:class="{ 'ml-3': isInLimitedRestoration }"
```

Reply:
Looks good, thanks.